src/neuroconv/datainterfaces/ecephys/intan/intandatainterface.py

```diff
@@ -8,35 +9,6 @@
 from ....utils import FilePathType, get_schema_from_hdmf_class
 
 
-def extract_electrode_metadata_with_pyintan(file_path) -> dict:
```

Comment:
I'm assuming this logic has propagated up?

Reply:
Yes. It should be fine on neo.